pkgs/by-name/re/recoverdm/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, installShellFiles
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "recoverdm";
+  version = "0.20";
+
+  src = fetchFromGitHub {
+    owner = "D3vil0p3r";
+    repo = "recoverdm";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-te/MCjY/O4Mq7n3ZxxvOb20S9usgPm5DVc6oqNXkizU=";
+  };
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  makeFlags = [
+    "-C src"
+  ] ++ lib.optional stdenv.isDarwin "LDFLAGS=-lcrypto";
+
+  postPatch = ''
+    mkdir -p $out/bin
+    substituteInPlace src/Makefile \
+      --replace /usr/local/bin $out/bin
+  '';
+
+  postInstall = ''
+    installManPage docs/recoverdm.1
+  '';
+
+  meta = with lib; {
+    description = "Recover damaged CD DVD and disks with bad sectors";
+    mainProgram = "recoverdm";
+    homepage = "https://github.com/D3vil0p3r/recoverdm";
```

Comment:
The upstream developer/maintainer of `recoverdm` is still active. Perhaps is @folkertvanheusden up to re-publish the source.

Reply:
Thank you for this reporting. If the original developer has a repository for it, I can change to it.

Reply:
can use https://salsa.debian.org/pkg-security-team/recoverdm ?
```nix
  version = "20-8";
  src = fetchFromGitLab {
    domain = "salsa.debian.org";
    owner = "pkg-security-team";
    repo = "recoverdm";
    rev = "debian/${finalAttrs.version}";
    hash = "sha256-1iW3Ug85ZLGpvG29N5zJt8oooSQGnLsr+8XIcp4aSSM=";
  };
```

Reply:
if there is no available primary source, i'd prefer using a well known secondary source, like https://salsa.debian.org/pkg-security-team/recoverdm.

Reply:
Fixed.